### ISSUE TITLE
should not check the vscode buitlin files

### ISSUE THREAD

**Issue by tjx666** [streetsidesoftware/vscode-spell-checker#2109]
![image](https://user-images.githubusercontent.com/41773861/181436991-475d44f9-f5a2-421c-8111-896ab04872a3.png)

Reproduce:

1. open a project contains `package.json`
2. open `package.json`

Versions:

```
Version: 1.70.0-insider (Universal)
Commit: b4d5b4ed69af953901051b1f049851b9dca3563e
Date: 2022-07-28T05:18:00.860Z
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Darwin x64 21.5.0
```

**Comment by Jason3S** [comment]
@tjx666,
Thank you. It will be fixed in the next release.

**Comment by Jason3S** [comment]
Fixed with v2.3.3
